contrib/face_recognition.lua

```diff
@@ -403,9 +414,9 @@ fc.height = dt.new_widget("entry"){
 
 fc.execute = dt.new_widget("button"){
   label = "detect faces",
-  clicked_callback = function(this) 
+  clicked_callback = function(this)
     face_recognition()
-  end
+end
```

Comment:
same as above

Reply:
I used an online beautifier to make sure all my changes were correctly indented. It also changed some of your code below “-- build the interface”. I will rollback it.